client/src/leap/soledad/client/api.py

```diff
@@ -70,6 +71,26 @@
 """
 SOLEDAD_CERT = None
 
+# A whitelist of modules from where to collect plugins dynamically.
+# For the moment restricted to leap namespace, but the idea is that we can pass
+# other "trusted" modules as options to the initialization of soledad.
```

Comment:
you're right, the soonest the better.
@andrejb @meskio how do you think it'd be better to pass that config variable?
maybe a simple .soledadrc?

config file, env variable, or initialization-time parameters. I think I'd go for the rc file for now.